index.js

```diff
@@ -52,6 +56,10 @@ function checkFragments(ast) {
       }
     }
   }
+
+  return assign({}, ast, {
```

Comment:
These are relatively simple operations, I'd really like to keep the `lodash` dependency out.